apps/docs/src/routes/+layout.svelte

```diff
@@ -45,36 +42,35 @@
   export let data: PageData
 </script>
 
-<svelte:window bind:innerWidth={window_width} />
 
 <PageTransition refresh={data.pathname}>
```

Comment:
not really related to these changes, but I would make the animation quicker (from 400 to 100, for example)